packages/crypto/src/lib.rs

```diff
@@ -2,7 +2,8 @@
 //! Please don't use any of these types directly, as
 //! they might change frequently, or be removed in the future.
 //! This crate does not adhere to semantic versioning.
-#![cfg_attr(feature = "backtraces", feature(backtrace))]
+#![cfg_attr(feature = "backtraces", feature(error_generic_member_access))]
+#![cfg_attr(feature = "backtraces", feature(provide_any))]
```

Comment:
I see you are putting into life those changes from `thiserror`: https://github.com/dtolnay/thiserror/pull/182/files